src/test/java/core/basesyntax/model/FruitRecordTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class FruitRecordTest {
+    private Fruit testFruit = new Fruit("test");
+    private FruitRecord fruitRecordExpected = new FruitRecord("b", testFruit, 10);
+
+    @Test
+    public void getOperation_Ok() {
+        String expectedOperation = fruitRecordExpected.getOperation();
+        FruitRecord test = new FruitRecord("b", new Fruit("test"), 10);
+        String actual = test.getOperation();
+        assertEquals(expectedOperation, actual);
+    }
+
+    @Test
+    public void getFruit() {
+        Fruit expected = fruitRecordExpected.getFruit();
+        FruitRecord test = new FruitRecord("b", new Fruit("test"), 10);
+        assertEquals(expected, test.getFruit());
+    }
+}
```

Comment:
I don`t have anything in FruitRecordFile to avoid negative value, which means i will need to rewrite code
I will checked with - FruitOperatorValidator